### ISSUE TITLE
Homepage template A11Y check

### ISSUE THREAD

**Issue by savitris** [frameless/developer.overheid.nl#103]
**Deze problemen kunnen op meerdere pagina's aanwezig zijn, waaronder:**
[Homepagina template](https://rijkshuisstijl-community-templates-3ix5516vd-nl-design-system.vercel.app/developer-overheid/homepage)

1. **Probleembeschrijving:**
De achtergrondkleuren van de navigatie-items in de hover-status lijken te veel op de witte achtergrondkleur van de webpagina (in Light-modus). De tekstkleur van het navigatie-item verandert niet. Hetzelfde geldt voor de rest van de elementen op de pagina, behalve voor de links in de footer.

**Referentiecode:**
`<a class="nl-link rhc-link rhc-nav-bar__link" href="/"><span class="rhc-nav-bar__label">Forum</span></a>`

**Gerelateerde WCAG:**
WCAG 1.4.3 Contrast Minimum
Level AA

**Referentieafbeelding:**
![Navigation item Forum is hovered over showing a very subtle background color change](https://github.com/user-attachments/assets/ce5c53b8-6134-4e4c-906a-3303cc397818)

_**Referentienummer:** TBD_

2. **Probleembeschrijving:**
De html lang-attribuut ontbreekt nog steeds.

**Referentiecode:**
`<html class="utrecht-root">`

**Gerelateerde WCAG:**
WCAG 3.1.1 Language of Page
Level A

**Referentieafbeelding:**

![Screenshot of the developer tools which show the page lacks a lang attribute](https://github.com/user-attachments/assets/87c118ac-8746-4caf-a7db-71b72f64b94e)

_**Referentienummer:** TBD_

3. **Probleembeschrijving:**
De vergrootglas-knop heeft geen toegankelijke naam.

**Referentiecode:**
`<button class="utrecht-button" type="button"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="tabler-icon tabler-icon-search "><path d="M10 10m-7 0a7 7 0 1 0 14 0a7 7 0 1 0 -14 0"></path><path d="M21 21l-6 -6"></path></svg></button>`

**Gerelateerde WCAG:**
WCAG 4.1.2 Name, Role, Value
Level A

**Referentieafbeelding:**
![The accessible names of the items in the main navigation are shown. The button with a magnifying glass icon has no accessible name.](https://github.com/user-attachments/assets/4337ffa0-8a74-4f1d-a8ab-43a454f32422)

_**Referentienummer:** TBD_

4. **Probleembeschrijving:**
Na het klikken op de "Terug naar boven" link, gaat de volgende focus-stop naar de Vercel-toolbar, en de focus daarna gaat naar de adresbalk van de browser. De focus na het klikken op de link zou naar de bovenkant van de webpagina moeten gaan.

**Referentiecode:**
`<a class="rhc-page-subfooter__backtotop" href="#">Terug naar boven <span class="utrecht-icon" role="presentation" aria-hidden="true"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="tabler-icon tabler-icon-arrow-narrow-up "><path d="M12 5l0 14"></path><path d="M16 9l-4 -4"></path><path d="M8 9l4 -4"></path></svg></span></a>`

**Gerelateerde WCAG:**
WCAG 2.4.3 Focus Order
Level A

**Referentieafbeelding:**
![A tab stop widget shows that after clicking the Terug naar boven link, the focus does not go to the top of the page](https://github.com/user-attachments/assets/fd7f19c4-81e2-4638-9318-7899db08acce)

_**Referentienummer:** TBD_

5. **Probleembeschrijving:**
De kaart-elementen hebben hun Header niet als een Heading 2, wat betekent dat de koptekst van elke kaart als gewone tekst wordt voorgelezen voor screenreadergebruikers.

**Referentiecode:**
`<div class="rhc-card__heading">Vind een repository</div>`

**Gerelateerde WCAG:**
WCAG 1.3.1 Info and Relationships
Level A

**Referentieafbeelding:**
![The code of a card element on the page shows it is not making the heading of the card into a heading element](https://github.com/user-attachments/assets/e47cc27b-b3b3-45db-830f-b40bcb377339)

_**Referentienummer:** TBD_

6. **Probleembeschrijving:**
De code van de kaart-elementen heeft nog steeds de afbeelding boven de Heading, waardoor screenreadergebruikers die item voor item door de pagina navigeren eerst de toegankelijke naam van de afbeelding horen vóór de Heading van de kaart.

**Referentiecode:**
```
<div class="rhc-card__content"><div class="rhc-card__image-container" data-testid="rhc-card__image-container"><img alt="Een beeldscherm met een HTML closing tag, symbool voor computer code" src="https://don.apps.digilab.network/img/computercode.svg" class="utrecht-img utrecht-img--photo rhc-card__image"></div><div class="rhc-card__icon"></div><div class="rhc-card__heading">Vind een repository</div><div class="rhc-card__description">Vind bestaande repositories om bij aan te haken en ontdek wat er binnen welke organisatie beschikbaar is.</div><div class="rhc-card__metadata"></div></div>
```

**Gerelateerde WCAG:**
WCAG 1.3.2 Meaningful Sequence 
Level A

**Referentieafbeelding:**
![the code of the one of the Card elements shows image is above the heading ](https://github.com/user-attachments/assets/9f09f1f5-ea9c-4cfc-841f-77a1d7f2c0a4)

_**Referentienummer:** TBD_

7. **Probleembeschrijving:**
Als de focus op de links van de kaart-elementen ligt, leest de screenreader de koptekst van de kaart en slaat de linktekst volledig over.

**Referentiecode:**
`<a class="nl-link rhc-link" href="#" aria-label="Vind een repository" title="Vind een repository">Ga naar het OSS register</a>`

**Gerelateerde WCAG:**
WCAG 2.4.4 Link Purpose
Level A

**Referentieafbeelding:**
![The code of the link of a card element shows its accessible name matches the card's heading and does not include the link text](https://github.com/user-attachments/assets/3b1aeb4c-53dc-464b-9171-29bd8fa89856)

_**Referentienummer:** TBD_
